sdk/go/common/util/gitutil/git.go

```diff
@@ -43,13 +43,13 @@ type VCSKind = string
 const (
 	defaultGitCloudRepositorySuffix = ".git"
 
-	// The host name for GitLab.
+	// GitLabHostName The host name for GitLab.
```

Comment:
I don't think we're consistent about this for values? I don't even the standard library bothers with repeating names for constant values?

Reply:
In goland it is recommended to add the variable name or function name at the front of the comment so I made these changes. (I think there will be such prompts in other IDEs as well)

Reply:
In fact, I think this code already explains what it is, and the comments here seem completely unnecessary.

Reply:
> I don't think we're consistent about this for values? I don't even the standard library bothers with repeating names for constant values?

Can I just remove these unnecessary comments?